pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1164 @@
+		"fsType":    "Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: \"ext4\", \"xfs\", \"ntfs\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+		"partition": "The partition in the volume that you want to mount. If omitted, the default is to mount by volume name. Examples: For volume /dev/sda1, you specify the partition as \"1\". Similarly, the volume partition for /dev/sda is \"0\" (or you can leave the property empty).",
+		"readOnly":  "Specify \"true\" to force and set the ReadOnly property in VolumeMounts to \"true\". If omitted, the default is \"false\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#awselasticblockstore",
+	}
+}
+
+func (Binding) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "Binding ties one object to another. For example, a pod is bound to a node by a scheduler.",
+		"metadata": "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"target":   "The target object that you want to bind to the standard object.",
+	}
+}
+
+func (Capabilities) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":     "Adds and removes POSIX capabilities from running containers.",
+		"add":  "Added capabilities",
+		"drop": "Removed capabilities",
+	}
+}
+
+func (ComponentCondition) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":        "ComponentCondition contains information for the condition of a component.",
+		"type":    "Type of component condition, currently only Healthy.",
+		"status":  "Current status of this component condition, one of True, False, Unknown.",
+		"message": "Health check message received from the component.",
+		"error":   "Error code from health check attempt (if any)",
+	}
+}
+
+func (ComponentStatus) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":           "ComponentStatus (and ComponentStatusList) holds the cluster validation info.",
+		"metadata":   "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"conditions": "List of component conditions observed",
+	}
+}
+
+func (ComponentStatusList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "ComponentStatusList is a list of ComponentStatus.",
+		"metadata": "Standard list metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds",
+		"items":    "List of component status objects.",
+	}
+}
+
+func (Container) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                       "Container represents a single container that is expected to be run on the host.",
+		"name":                   "Name of the container. This must be a DNS_LABEL. Each container in a pod must have a unique name. Cannot be updated.",
```

Comment:
Revision:
"Name of the container specified as a DNS_LABEL. Each container in a pod must have a unique name (DNS_LABEL). This property cannot be updated after the object is created."

@bgrant0607 Is this revision correct, especially the "cannot be updated" part?